src/tests/build-test.js

```diff
@@ -61,4 +74,5 @@ function simpleTest(url) {
 }
 
 //simpleTest('/fishes.jpg')
-saveInc('/fishes.jpg', glitchParams)
+//saveInc('/fishes.jpg', glitchParams)
+simpleGm()
```

Comment:
with the simpleGm() function is completely successfull:

```
Run npm run build-test --if-present
  npm run build-test --if-present
  shell: /usr/bin/bash -e {0}

> visual-ar-poetry-backend@0.1.0 build-test /home/runner/work/visualARPoetry-backend/visualARPoetry-backend
> node src/tests/build-test.js

/home/runner/work/visualARPoetry-backend/visualARPoetry-backend/src/tests/../imgs/test.jpg created  :: gm "convert" "-size" "560x110" "xc:#00ff55aa" "-pointsize" "68" "-strokewidth" "2" "-stroke" "#efe" "-fill" "#555" "-draw" "text 20,72 \"Visual\"" "-fill" "#fa0" "-draw" "text 232,72 \"AR Poetry\"" "/home/runner/work/visualARPoetry-backend/visualARPoetry-backend/src/tests/../imgs/test.jpg"
```
https://github.com/kalwalt/visualARPoetry-backend/runs/2333346994?check_suite_focus=true